R/add_pop.R

```diff
@@ -77,6 +77,9 @@ gs_add_gating_method <- function(gs, alias = "*"
     preprocessing_args <- .argDeparser(preprocessing_args)      
   }
   
+  # format split dims & drop duplicated dims
+  dims <- paste(unique(dims), collapse = ",")
```

Comment:
Is it OK if dims is more than 2-D?

Reply:
openCyto only supports 1D or 2D gating, but yes it will work with more than two dims.

Reply:
https://github.com/RGLab/openCyto/blob/323430ebb251e69db7f694751f2a9a2e5cba4d26/R/csvTemplate-parser.R#L291-L296

Reply:
I would love for us to support multiple dims (CytoExploreR uses a hacky way to get around this) but the issue is that the gating functions within openCyto don't perform a dims check so we'd need to update all of them to ensure that appropriate dimensions are supplied for each of the supported gating functions.

Reply:
We could possibly check the dims against the list of supported gating methods but that will have to be updated every time we add a new gating method.